app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ArticlesController < ApplicationController
```

Comment:
How could the articles be protected from deletion/modification by logged-in non-admin users?